schemas/answers/address.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "https://eq.ons.gov.uk/answers/address.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "answer": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+      },
+      "q_code": {
```

Comment:
q_code? Are we supporting business on this one

Reply:
Runner still support data version 0.0.1 hence `q_code` is kept.